tests/build_scripts/test_validate_plugin_version_bump.py

```
@@ -0,0 +1,237 @@
+    assert len(errs) == 1
+    assert CLAUDE in errs[0]
+
+
+def test_missing_current_version_is_config_error():
+    v, errs = vpb.evaluate([".claude/x.md"], _pairs(claude=("0.3.0", None)))
+    assert v == []
+    assert len(errs) == 1
+
+
+def test_backslash_paths_normalized():
+    v, errs = vpb.evaluate(
```

Ruff (0.15.14)

[warning] Unpacked variable `errs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)